scalafix/rules/src/main/scala/rsc/rules/RscCompat.scala

```diff
@@ -11,39 +11,39 @@ import rsc.rules.syntax._
 import scala.meta._
```

Comment:
Please split the pull request into several commits: 1) update to v2, 2) add unit tests, 3) fix the broken integration test. Current arrangement makes it hard to review these changes in isolation.

Reply:
I've gotten used to phablet style squashing everything down. I will split them up tomorrow.